addons/mass_mailing/static/src/themes/theme_selector/theme_selector.xml

```diff
@@ -1,21 +1,37 @@
 <templates id="template">
     <div t-name="mass_mailing.ThemeSelector" class="o_mail_theme_selector">
```

Comment:
seems like this element has a bigger height than its parent when the window has a width of approx. 1000px, before it was not an issue since the parent had a margin/padding at the bottom, but since you changed things according to that, the theme_selector overflows below its parent and the overflowing part is not visible.